packages/http-specs/specs/client/structure/client-operation-group/client.tsp

```diff
@@ -0,0 +1,74 @@
+import "./main.tsp";
```

Comment:
Client folder also needed to go
There should not me any import for @azure-tools

Reply:
Deleted the client folder.